src/Hazelcast.Net/Clustering/ClusterConnections.cs

```diff
@@ -835,6 +835,9 @@ private async Task<MemberConnection> ConnectAsync(NetworkAddress address, Networ
             // connect to the server (may throw and that is ok here)
             var result = await connection.ConnectAsync(_clusterState, cancellationToken).CfAwait();
 
+            // we now that fail over is an ee feature.
+            _clusterState.IsEnterprise = result.FailoverSupported;
```

Comment:
In the Java client, we send servers whether or not we are EE [as a client attribute](https://github.com/hazelcast/hazelcast-mono/blob/master/hazelcast/hazelcast/src/main/java/com/hazelcast/client/impl/ClientEndpointImpl.java#L169), and internally we are also [able to query `BuildInfo` to check if we are EE](https://github.com/hazelcast/hazelcast-mono/blob/master/hazelcast/hazelcast/src/main/java/com/hazelcast/client/impl/clientside/DefaultClientExtension.java#L200) - is there nothing similar on .NET client?

Reply:
Yes, in Java client there are options. However, except Java, other clients have no EE version. This is the best effort for EE check.